FrontEnd/src/components/Content.js

```diff
@@ -53,9 +54,13 @@ class Content extends React.Component {
         return (
             <div className={"content"}>
                 <div className="book-card-wrapper">
+                    <Row>
                     {this.state.bookCardsArray.map(b => (
-                        <BookCard key={b.id} bookID={b.id} bookTitle={b.value.title} bookAuthors={b.value.authors} bookStatus={b.value.status} />
+                        <Col xl={3} l={4} m={6} s={12} key={b.id}>
```

Comment:
have you tried this with opening and closing the filter bar? does it work with this?
I've started out in a similar way, but then the filter bar ended up squishing things, hence I dropped all this and went with the flex-box solution